tests/integration/targets/ec2_snapshot/tasks/main.yml

```diff
@@ -217,6 +217,61 @@
         that:
           - info_result.snapshots| length == 3
 
+    # check that snapshot_ids and max_results are mutually exclusive
+    - name: Get info about all snapshots in paginated mode
```

Comment:
The names of the new tasks should reflect what they are testing - in CI results we won't be able to see the code comments.

Reply:
> The names of the new tasks should reflect what they are testing - in CI results we won't be able to see the code comments.

sure, I will update it
Thanks